packages/preview-middleware-client/.eslintrc.js

```diff
@@ -21,10 +21,7 @@ module.exports = {
                 varsIgnorePattern: '^_',
                 argsIgnorePattern: '^_'
             }
-        ],
-        '@typescript-eslint/no-unsafe-argument': 'warn',
-        '@typescript-eslint/no-unsafe-member-access': 'warn',
-        '@typescript-eslint/no-unsafe-assignment': 'warn'
```

Comment:
@devinea I think we should keep the rules as they are and work on fxing them. We could consider relaxing the rules for test cases, like done on root: https://github.com/SAP/open-ux-tools/blob/b5277ea58890472c7aab0950a6deb8e101db1730/.eslintrc#L59-L108

Reply:
Reverted in https://github.com/SAP/open-ux-tools/pull/1909/commits/46d7ef589df4cf1aae04ec91bd9cb58e4fb37c5e